### ISSUE TITLE
difftool

### ISSUE THREAD

**Issue by bliednov** [nvie/gitflow#315]
What about adding this possibility?
It would be nice if I could use my difftool, which is configure in .gitconfig.

``` bash
git flow feature difftool
```

**Comment by deiga** [comment]
I don't quite understand the use case here. Can't you just do `git diff feature/foo develop`?

**Comment by bliednov** [comment]
Yeah, you could. You even could do `git difftool feature/foo develop` to use e.g. kdiff.
The thing is that gitflow already has `git flow feature diff` and I'd like to have a possibility to use `difftool`.

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
